ddln_imaginator.proto

```diff
@@ -21,6 +21,64 @@ enum ImageFormat {
     GRAY = 4;
 }
 
+message Environment {
+    message Clouds {
+        enum Type {
+            NONE = 0;
+            ALTOCUMULUS = 1;
+            CLOUDS_BASE = 2;
+            CUMULUS = 3;
+            NIMBOSTRATUS = 4;
+            STRATOCUMULUS = 5;
+            STRATUS = 6;
+        }
+        optional Type type = 1;
+        optional float altitude_m = 2;
```

Comment:
I would expect min/max altitude for layers, also I expect to see some density parameter

Reply:
Good question. Not sure about the answer but let's follow Unigine API here.

Reply:
Unigine supports any altitude, so I would give more freedom for the user.
Density is related with the cloud types. I'm not sure if it will keep the type in that case.

Reply:
Density option will be added.